src/GafferScene/DeleteCurves.cpp

```diff
@@ -113,9 +113,7 @@ IECore::ConstObjectPtr DeleteCurves::computeProcessedObject( const ScenePath &pa
 
 	std::string deletePrimVarName = curvesPlug()->getValue();
 
-	/// \todo Remove. We take values verbatim everywhere else in Gaffer, and I don't
```

Comment:
I think the reason I didn't do this at the time of adding the todo was that it's technically a change in behaviour. Can we target this one commit to master instead?

Reply:
I dropped this commit and moved it to master in #4069